### PR TITLE
Add jsdoc plugin and jsdoc rules.

### PR DESCRIPTION
**Pull request by aljones15**
fixes eslint stuff to work with latest release.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ module.exports = {
   env: {
     browser: true
   },
-  plugins: ['jsdoc'],
-  extends: 'eslint-config-digitalbazaar',
+  extends: [
+    'eslint-config-digitalbazaar',
+    'eslint-config-digitalbazaar/doc'
```
Comment (mattcollier) [inline .eslintrc.js]: it's `/jsdoc` isn't it?

**Review by mattcollier — CHANGES_REQUESTED** [review]

**Comment by mattcollier** [comment]
Also, you need to bump the `eslint-config-digitalbazaar` dep to 2.0.0, correct?

**Comment by mattcollier** [comment]
@aljones15 I don't know what you've done to create the commit `Merge branch 'master' into upgrade-eslint` but it needs to be removed. I would expect that this branch is already based on master, so I don't know what is going on here.

**Comment by aljones15** [comment]
> @aljones15 I don't know what you've done to create the commit `Merge branch 'master' into upgrade-eslint` but it needs to be removed. I would expect that this branch is already based on master, so I don't know what is going on here.

squash should turn it into one commit. what you are requesting requires an entire rebase and rewriting an entire branch. this was branched from master, but the changes to the package.json file caused a conflict.

ok got it apparently I had not pulled master recently.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**